### PR TITLE
Msw 119023095 email spec

### PR DESCRIPTION
**Pull request by michaelwehrley**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,7 +5,9 @@ group :development, :test do
 end
 
 group :test do
-  gem "rspec", "~> 3.4"
   gem "capybara", "~> 2.1"
+  gem "jasmine", "~> 2.4"
+  gem "jshintrb", "~> 0.2"
```
Comment (ericcf) [inline Gemfile]: I'd prefer to standardize on eslint going forward, but don't worry about switching for this project.

Comment (michaelwehrley) [inline Gemfile]: I had looked at `social_networking`.  I should have looked more.

**Comment by michaelwehrley** [comment]
@ericcf I know there are some commented code that isn't being used.  I can make that a chore later if necessary.

**Comment by ericcf** [comment]
Up to you. I would just delete any commented code: you know for sure it isn't needed by the app because it's not being run!

File: js/app/app.js [hunk 2]
```diff
@@ -936,13 +936,28 @@ app.actions.showSlideShow = function (event) {
...
 app.actions.sendEmail = function(data){
 	$.ajax({
-	  type: "POST",
-	  url: "http://mohrlab.northwestern.edu/utility/sendmail.cfm",
-	  data: { mailJSON: JSON.stringify(data) },
-	  success: function(data, textStatus, jqXHR) {
-	  	alert("Email was sent!");
-	  }
+		type: "POST",
+		url: "http://mohrlab.northwestern.edu/utility/sendmail.cfm",
```
Comment (ericcf) [inline js/app/app.js]: I think this needs to be `https`, but you'll have to test the email functionality anyway to make sure.

**Pull request merged**